src/util.f90

```diff
@@ -0,0 +1,32 @@
+module util
+    implicit none
+
+contains
+
+    subroutine arange(a, b, dx, x)
+        real(8), intent(in) :: a, b, dx
+        real(8), dimension(:), intent(out) :: x
+
+        integer :: i, n
+
+        n = int((b - a) / dx) + 1
+        do i = 1, n
+            x(i) = a + (i - 1) * dx
+        end do
+    end subroutine arange
+
+    subroutine linspace(a, b, n, x)
```

Comment:
Might I interest you in a [`linspace`](../../blob/main/src/magfie/coil_tools.f90#L28) that also covers e.g. half-open intervals like [0, 2π]?

Reply:
please move it here!

Reply:
have moved it here